lit_gpt/utils.py

```diff
@@ -371,6 +373,20 @@ def __iter__(self) -> Self:
         return self
 
 
+def copy_config_files(source_dir: Path, out_dir: Path) -> None:
+    """Copies the specified configuration and tokenizer files into the output directory."""
+
+    # TODO: All the config files need to be present, should we raise an error if one is missing?
```

Comment:
```suggestion
```

Reply:
Oh yeah, forgot about this.
We can still discuss whether we want to have warnings/errors. 

I think warnings would be ok. Errors I'm not sure, because it could be that the user is training with a custom pretrained checkpoint that doesn't have these files. But no strong opinions about this. @carmocca ?

Reply:
I agree. We cannot reliably know this if this is used in any training script. I wouldn't add warnings or errors